src/messages/GE.json

```diff
@@ -498,5 +498,10 @@
   "Revoking admin granted scopes": "Revoking admin granted scopes",
   "Revoking default scopes": "Revoking default scopes",
   "More request area items": "More request area items",
-  "More response items": "More response area items"
+  "More response items": "More response area items",
+  "Banner notification 1 header": "New to Graph Explorer?",
+  "Banner notification 1 content": "Graph Explorer is a developer tool that let's you learn about Microsoft Graph APIs. Use Graph Explorer to try the APIs on the default sample tenat to explore capabilities.",
+  "Banner notification 1 link text": "Follow a step-by-step tutorial",
+  "Banner notification 1 link": "https://aka.ms/ge",
```

Comment:
@jessicaaawu I'll need a URL for the tutorial

Reply:
Hey @musale , please use this link https://learn.microsoft.com/en-us/graph/graph-explorer/graph-explorer-overview?view=graph-rest-1.0